abdm_python_integrator/abha/utils/abha_creation_util.py

```diff
@@ -0,0 +1,44 @@
+from abdm_python_integrator.abha.const import (
+    CREATE_HEALTH_ID_URL,
+    GENERATE_AADHAAR_OTP_URL,
+    GENERATE_MOBILE_OTP_URL,
+    VERIFY_AADHAAR_OTP_URL,
+    VERIFY_MOBILE_OTP_URL,
+)
+from abdm_python_integrator.utils import ABDMRequestHelper
+
+
+def generate_aadhaar_otp(aadhaar_number):
+    payload = {"aadhaar": str(aadhaar_number)}
+    return ABDMRequestHelper().abha_post(GENERATE_AADHAAR_OTP_URL, payload)
+
+
+def generate_mobile_otp(mobile_number, txnid):
+    payload = {"mobile": str(mobile_number), "txnId": txnid}
+    return ABDMRequestHelper().abha_post(GENERATE_MOBILE_OTP_URL, payload)
```

Comment:
Since we are implementing this as a pluggable app interfacing with ABHA, should we catch these exceptions and (create and) raise corresponding exceptions (and document them in this app)?

Reply:
Yes definitely.
I did not included them in the current PR and created a separate ticket as per [comment](https://github.com/dimagi/abdm-python-integrator/pull/1#issuecomment-1715068793) for reasons mentioned below:

- It will require deciding on a error format for client end for M1.
- Probably requre modifications on the mobile app side 
- Looks better to take it up separately looking at scope and majorly to avoid impact on M2/M3 work.

> NOTE: M2/M3 implementation which requires gateway call has a different response format will have these exceptions handled.

Hope that works.

Reply:
Cool, that sounds good.